mne/io/edf/edf.py

```diff
@@ -681,46 +681,29 @@ def _get_info(
         info["subject_info"]["weight"] = float(edf_info["subject_info"]["weight"])
 
     # Filter settings
-    highpass = edf_info["highpass"]
-    lowpass = edf_info["lowpass"]
-    if highpass.size == 0:
-        pass
-    elif all(highpass):
-        if highpass[0] == "NaN":
-            # Placeholder for future use. Highpass set in _empty_info.
-            pass
-        elif highpass[0] == "DC":
-            info["highpass"] = 0.0
+    if "highpass" in edf_info and len(edf_info["highpass"]):
```

Comment:
More compact equivalent if you want it
```suggestion
    if len(edf_info.get("highpass", [])):
```

Reply:
Thanks for the tips.
I updated these codes for GDF with `_set_prefilter` function
and similar code was implemented there.